crates/db_schema/src/newtypes.rs

```diff
@@ -420,3 +422,31 @@ impl InstanceId {
 #[cfg_attr(feature = "full", ts(export))]
 /// The internal tag id.
 pub struct TagId(pub i32);
+
+/// A pagination cursor
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+pub struct PaginationCursor(pub String);
+
+#[cfg(feature = "full")]
+impl PaginationCursor {
+  pub fn create(prefix: char, id: i32) -> Self {
```

Comment:
This could be called `new` instead since it's essentially a constructor.

Reply:
Makes sense.